### PR TITLE
Add wip category and application needle

### PR DESCRIPTION
**Pull request by MitraDarja**
Adds a Coming Soon section to the application page and adds the wip app needle

### CODE REVIEW

File: 04_applications.md [hunk 1]
```diff
@@ -19,8 +20,16 @@ The permissive open source BSD-license of the library enables you to integrate S
 attribution being the only legal requirement. But, please also cite our most recent SeqAn publication when you use SeqAn
 in your academic work.
 
-### Official
 
+{% for category in page.categories %}
+{%- if category == "official" -%}
+### Official
+{% else %}
+### Comming Soon
```
Comment (eseiler) [inline 04_applications.md:28]:
```suggestion
### Coming Soon
```

File: _apps/needle.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+algorithms and data structures. For instance, at the moment it is not possible to find all experiments in the SRA
+containing certain genes with a specified expression profile. But if the data can not be found in a meaningful way,
+the information is lost despite the huge affords made to store these experiments.
+
+**Tool Description**: Needle is a tool for storing sequencing experiments in such a way that approximate quantification
+of large sequencing data sets is possible.
+
+Needle is based on the Interleaved Bloom Filter (IBF) and its basic idea is to store multiple IBFs for different
+expression levels.
+
```
Comment (eseiler) [inline _apps/needle.md:21]: Can you do a line length of 120?

**Comment by eseiler** [comment]
Did you rebase on master? I think you are missing some stuff (e.g., on master the citation doesn't show an error if you do not provide a citation)

**Comment by MitraDarja** [comment]
@eseiler Thanks for the tip. :)

**Review by marehr — APPROVED** [review]

**Comment by marehr** [comment]
ty ❤️

**Pull request merged**